creator/management/commands/c2m2_load.py

```diff
@@ -0,0 +1,358 @@
+import os
+import csv
+import logging
+import requests
+from django.core.management.base import BaseCommand, CommandError
+
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+ROOT_PROJECT_NS = "kidsfirst:"
+ROOT_PROJECT_LOCAL_ID = "drc"
+ROOT_PROJECT_ABBR = "KFDRC"
+ROOT_PROJECT_NAME = (
+    "The Gabriella Miller Kids First Pediatric Research Program"
+)
+ROOT_PROJECT_DESCRIPTION = """
+A large-scale data resource to help researchers uncover new insights into the 
+biology of childhood cancer and structural birth defects.
+""".replace(
+    "\n", ""
+)
+ROOT_PROJECT_URL = "https://kidsfirstdrc.org"
+
+CONTACT_EMAIL = "support@kidsfirstdrc.org"
+CONTACT_NAME = "Kids First Support"
+
+# This is the prefix used for each sub-project in Kids First (Each study)
+PROJECT_PREFIX = "Kids First: "
+
+# This is a list of studies that are currently on the portal
+# We will only add these studies to the submission by default
+PUBLISHED_STUDIES = [
+    "SD_DK0KRWK8",
+    "SD_NMVV8A1Y",
+    "SD_R0EPRSGS",
+    "SD_7NQ9151J",
+    "SD_YNSSAPHE",
+    "SD_9PYZAHHE",
+    "SD_DZ4GPQX6",
+    "SD_W0V965XZ",
+    "SD_ZXJFFMEF",
+    "SD_M3DBXD12",
+    "SD_8Y99QZJJ",
+    "SD_46RR9ZR6",
+    "SD_PREASA7S",
+    "SD_B8X3C1MX",
+    "SD_BHJXBDQK",
+    "SD_1P41Z782",
+    "SD_DYPMEHHF",
+    "SD_RM8AFW0R",
+    "SD_YGVA0E1C",
+    "SD_6FPYJQBR",
+    "SD_ZFGDG5YS",
+    "SD_46SK55A3",
+]
+
+
+class Command(BaseCommand):
+    help = "Create C2M2 Level 0 Manifest from Dataservice"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--studies",
+            type=str,
+            default=PUBLISHED_STUDIES,
+            nargs="*",
+            help="kf_ids of studies to include in the submission file",
+        )
+        parser.add_argument(
+            "--out-dir",
+            type=str,
+            default="c2m2_submission",
+            help="The directory to place the submission files in",
+        )
+
+    def handle(self, *args, **options):
+        logger.info(
+            f"Will build a submission file for {len(options.get('studies'))} "
+            "studies."
+        )
+        self.out_dir = options.get("out_dir")
+        self.setup_directory()
+
+        # By default we will process all published studies
+        self.studies = self.get_studies(options.get("studies"))
```

Comment:
In case you later remove the default value for studies option
```suggestion
        self.studies = self.get_studies(options.get("studies", []))
```